docs/src/docs/cli/installation.md

```diff
@@ -50,7 +50,7 @@ There are two generations of Mac hardware, if you aren't sure which you have [ch
   Apple Silicon / ARM64
 </a>
 
-2. MacOS: <a href="https://github.com/jozu-ai/kitops/releases/latest/download/kitops-darwin-x86_64.zip"
+2. MacOS: <a href="https://kitops.gateway.scarf.sh/github/macos/intesl_x86_64/latest"
```

Comment:
Just double checking for a typo here: `intesl_x86_64`?

Reply:
yeah, i check that like twice and its working for me haha so i guess @Jwilliamsr (or somebody else?) made a typo when creating the links...